src/main/java/net/atos/zac/aanvraag/ProductaanvraagService.java

```diff
@@ -195,11 +210,23 @@ private void pairAanvraagPDFWithZaak(final ProductaanvraagDenhaag productaanvraa
         final ZaakInformatieobject zaakInformatieobject = new ZaakInformatieobject();
         zaakInformatieobject.setInformatieobject(productaanvraag.getPdfUrl());
         zaakInformatieobject.setZaak(zaakUrl);
-        zaakInformatieobject.setTitel(ZAAK_INFORMATIEOBJECT_TITEL);
-        zaakInformatieobject.setBeschrijving(ZAAK_INFORMATIEOBJECT_BESCHRIJVING);
+        zaakInformatieobject.setTitel(AANVRAAG_PDF_TITEL);
+        zaakInformatieobject.setBeschrijving(AANVRAAG_PDF_BESCHRIJVING);
         zrcClientService.createZaakInformatieobject(zaakInformatieobject, ZAAK_INFORMATIEOBJECT_REDEN);
     }
 
+    private void pairBijlagenWithZaak(final ProductaanvraagDenhaag productaanvraag, final URI zaakUrl) {
```

Comment:
Hier is niet de hele productaanvraag nodig. Alleen de attachements doorgeven is genoeg.